### ISSUE TITLE
NullPointerException createDisguise on itemframe disguise.

### ISSUE THREAD

**Issue by mibby** [libraryaddict/LibsDisguises#139]
LibsDisguise dev 36
Spigot 1.10

```
[01:31:11] [Server thread/INFO]: mibby issued server command: /dis itemframe
[01:31:11] [Server thread/INFO]: java.lang.NullPointerException
[01:31:11] [Server thread/INFO]:    at me.libraryaddict.disguise.disguisetypes.Disguise.createDisguise(Disguise.java:102)
[01:31:11] [Server thread/INFO]:    at me.libraryaddict.disguise.disguisetypes.MiscDisguise.<init>(MiscDisguise.java:33)
[01:31:11] [Server thread/INFO]:    at me.libraryaddict.disguise.commands.BaseDisguiseCommand.parseDisguise(BaseDisguiseCommand.java:638)
[01:31:11] [Server thread/INFO]:    at me.libraryaddict.disguise.commands.DisguiseCommand.onCommand(DisguiseCommand.java:28)
[01:31:11] [Server thread/INFO]:    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[01:31:11] [Server thread/INFO]:    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:152)
[01:31:11] [Server thread/INFO]:    at org.bukkit.craftbukkit.v1_10_R1.CraftServer.dispatchCommand(CraftServer.java:665)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PlayerConnection.handleCommand(PlayerConnection.java:1386)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1191)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:5)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13)
[01:31:11] [Server thread/INFO]:    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[01:31:11] [Server thread/INFO]:    at java.util.concurrent.FutureTask.run(Unknown Source)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:793)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:403)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:730)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:629)
[01:31:11] [Server thread/INFO]:    at java.lang.Thread.run(Unknown Source)
[01:31:11] [Server thread/INFO]: java.lang.NullPointerException
[01:31:11] [Server thread/INFO]:    at me.libraryaddict.disguise.commands.BaseDisguiseCommand.parseDisguise(BaseDisguiseCommand.java:646)
[01:31:11] [Server thread/INFO]:    at me.libraryaddict.disguise.commands.DisguiseCommand.onCommand(DisguiseCommand.java:28)
[01:31:11] [Server thread/INFO]:    at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[01:31:11] [Server thread/INFO]:    at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:152)
[01:31:11] [Server thread/INFO]:    at org.bukkit.craftbukkit.v1_10_R1.CraftServer.dispatchCommand(CraftServer.java:665)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PlayerConnection.handleCommand(PlayerConnection.java:1386)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:1191)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PacketPlayInChat.a(PacketPlayInChat.java:5)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13)
[01:31:11] [Server thread/INFO]:    at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[01:31:11] [Server thread/INFO]:    at java.util.concurrent.FutureTask.run(Unknown Source)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:793)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:403)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:730)
[01:31:11] [Server thread/INFO]:    at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:629)
[01:31:11] [Server thread/INFO]:    at java.lang.Thread.run(Unknown Source)
```

**Comment by libraryaddict** [comment]
No idea if the disguise is fixed.

**Comment by Krowerom** [comment]
I don't think it is fixed yet - I get the same crash on 1.10 running 9.0.7 when using the Falling_block or Item_Frame disguises.  Great job getting the polarbear, husk and .. er.. new skeleton thingy in though :D
